src/data/ACTIONS/layers/patch6.1.ts

```diff
@@ -9,5 +10,11 @@ export const patch610: Layer<ActionRoot> = {
 		GRIT: {cooldown: 3000},
 		IRON_WILL: {cooldown: 3000},
 		ROYAL_GUARD: {cooldown: 3000},
+
+		//SAM 6.1 action changes:
+		HISSATSU_KAITEN: {id: SHARED.UNKNOWN.id}, //Kaiten was removed. But is job critical for pre-6.1 analysis.
```

Comment:
Not sure why you only wanna kill the ID instead of the whole status here.
```suggestion
		HISSATSU_KAITEN: SHARED.UNKNOWN,
```